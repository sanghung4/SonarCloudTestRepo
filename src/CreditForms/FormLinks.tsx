import { useContext } from 'react';

import {
  Box,
  Grid,
  List,
  ListItem,
  ListItemIcon,
  ListItemText
} from '@dialexa/reece-component-library';

import { SavePdfIcon } from 'icons';
import { useDomainInfo } from 'hooks/useDomainInfo';
import { AuthContext } from 'AuthProvider';
import { ErpSystemEnum } from 'generated/graphql';
import { useSelectedAccountsContext } from 'providers/SelectedAccountsProvider';

interface CreditFormURLType {
  url: string;
  title: string;
  authorizationRequired?: boolean;
}

const commmonFormURLS: CreditFormURLType[] = [
  {
    url: 'MORSCO-Credit-Application-All-Brands-January-2023.pdf',
    title: 'Customer Account Credit Application'
  },
  {
    url: 'MORSCO-Credit-Application-All-Brands-Spanish-January-2023.pdf',
    title: 'Solicitud de Crédito'
  },
  {
    url: 'INVOICE-AND-DELIVERY-TICKET-Terms-of-Sale_12-29-2014.pdf',
    title: 'Invoice & Delivery Ticket Terms of Sale'
  },
  {
    url: 'EFT-Customer-Authorization-Form-Morrison.pdf',
    title: 'Electronic Funds Transfer Authorization Form'
  },
  {
    url: 'Morsco_Tool_Rental_Agreement_Form_20211105.pdf',
    title: 'Equipment Rental Agreement',
    authorizationRequired: true
  }
];

const fortilineFormURLS: CreditFormURLType[] = [
  {
    url: 'MORSCO-Credit-Application-All-Brands-January-2023.pdf',
    title: 'Customer Account Credit Application'
  },
  {
    url: 'MORSCO-Credit-Application-All-Brands-Spanish-January-2023.pdf',
    title: 'Solicitud de Crédito'
  },
  {
    url: 'EFT-One-Time-Form-WW-V2.pdf',
    title: 'Electronic Funds Transfer Authorization Form'
  }
];

function FormLinks() {
  const { isWaterworks } = useDomainInfo();
  const { authState } = useContext(AuthContext);
  const { selectedAccounts } = useSelectedAccountsContext();

  const isMincron = selectedAccounts.erpSystemName === ErpSystemEnum.Mincron;
  const creditFormURLS: CreditFormURLType[] =
    isMincron || isWaterworks ? fortilineFormURLS : commmonFormURLS;

  return (
    <Grid container>
      <Grid item xs={12} md={4}>
        <List>
          {creditFormURLS
            .filter(
              (url) => !url.authorizationRequired || authState?.isAuthenticated
            )
            .map((item, i, arr) => (
              <ListItem
                key={item.title}
                component="a"
                href={`/files/credit-forms/${item.url}`}
                target="_blank"
                divider={i < arr.length - 1}
                disableGutters
              >
                <ListItemIcon>
                  <SavePdfIcon />
                </ListItemIcon>
                <ListItemText
                  primary={
                    <Box component="span" color="primary02.main">
                      {item.title}
                    </Box>
                  }
                />
              </ListItem>
            ))}
        </List>
      </Grid>
    </Grid>
  );
}
export default FormLinks;
