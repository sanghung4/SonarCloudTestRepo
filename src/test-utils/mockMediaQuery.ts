import mediaQuery from 'css-mediaquery';

function createMatchMedia(width: number) {
  return (query: string): MediaQueryList => ({
    matches: mediaQuery.match(query, { width }),
    media: query,
    onchange: null,
    addListener: () => jest.fn(),
    removeListener: () => jest.fn(),
    addEventListener: jest.fn(),
    removeEventListener: jest.fn(),
    dispatchEvent: jest.fn()
  });
}

export const breakpoints = {
  mobile: 414,
  desktop: 1200
};

export const setBreakpoint = (breakpoint: 'mobile' | 'desktop' = 'desktop') => {
  Object.defineProperty(window, 'innerWidth', {
    writable: true,
    configurable: true,
    value: breakpoints[breakpoint]
  });
  window.dispatchEvent(new Event('resize'));

  window.matchMedia = createMatchMedia(window.innerWidth);
};
