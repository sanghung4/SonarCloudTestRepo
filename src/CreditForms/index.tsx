import { useTranslation } from 'react-i18next';
import Breadcrumbs from 'common/Breadcrumbs';

import {
  Box,
  Card,
  Container,
  Link,
  Typography,
  useScreenSize
} from '@dialexa/reece-component-library';

import useDocumentTitle from 'hooks/useDocumentTitle';
import FormLinks from './FormLinks';

function CreditForms() {
  const { t } = useTranslation();
  useDocumentTitle(t('common.creditAndForms'));
  const { isSmallScreen } = useScreenSize();

  return (
    <>
      <Breadcrumbs
        pageTitle={t('common.creditForms')}
        config={[
          {
            text: t('common.resources')
          }
        ]}
      />
      {/* Header Section */}
      <Container>
        {/* Filters Section */}
        <Card
          sx={{
            px: isSmallScreen ? 3 : 5,
            py: isSmallScreen ? 3 : 3,
            mb: 3
          }}
        >
          <Typography variant="h5">{t('common.creditForms')}</Typography>
        </Card>
        <Card
          sx={{
            px: isSmallScreen ? 2 : 5,
            py: isSmallScreen ? 2 : 3,
            mb: 3
          }}
        >
          <Box pb={3}>
            <Typography variant="body1" display="inline">
              {t('creditForms.info1')}
            </Typography>
            <Link href={`mailto:${t('creditForms.email')}`}>
              <Typography
                variant="body1"
                display="inline"
                sx={{ color: 'primary02.main' }}
              >
                {t('creditForms.email')}
              </Typography>
            </Link>
            <Typography variant="body1" display="inline">
              {t('creditForms.info2')}
            </Typography>
          </Box>
          <FormLinks />
        </Card>
      </Container>
    </>
  );
}

export default CreditForms;
