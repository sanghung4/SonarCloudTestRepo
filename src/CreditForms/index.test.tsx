import CreditForms from 'CreditForms';
import { setBreakpoint } from 'test-utils/mockMediaQuery';
import { render } from 'test-utils/TestWrapper';
import { useDomainInfo } from 'hooks/useDomainInfo';

const mockDomainInfo: ReturnType<typeof useDomainInfo> = {
  companyList: [],
  companyNameLink: '',
  companyNameList: '',
  engine: '',
  isWaterworks: false,
  rootDomain: '',
  subdomain: '',
  port: ''
};

jest.mock('hooks/useDomainInfo', () => {
  const originalModule = jest.requireActual('hooks/useDomainInfo');
  return {
    __esModule: true,
    ...originalModule,
    useDomainInfo: () => mockDomainInfo
  };
});

describe('CreditForms', () => {
  it('CreditForms should be matching the snapshot', () => {
    setBreakpoint('desktop');
    mockDomainInfo.isWaterworks = false;
    const { container } = render(<CreditForms />);
    expect(container).toMatchSnapshot();
  });

  it('CreditForms for fortiline should be matching the snapshot', () => {
    setBreakpoint('desktop');
    mockDomainInfo.isWaterworks = true;
    const { container } = render(<CreditForms />);
    expect(container).toMatchSnapshot();
  });

  it('CreditForms should be matching the snapshot on mobile', () => {
    setBreakpoint('mobile');
    mockDomainInfo.isWaterworks = false;
    const { container } = render(<CreditForms />);
    expect(container).toMatchSnapshot();
  });

  it('CreditForms for fortiline should be matching the snapshot on mobile', () => {
    setBreakpoint('mobile');
    mockDomainInfo.isWaterworks = true;
    const { container } = render(<CreditForms />);
    expect(container).toMatchSnapshot();
  });
});
