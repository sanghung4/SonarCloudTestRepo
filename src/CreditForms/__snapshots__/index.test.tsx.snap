// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditForms CreditForms for fortiline should be matching the snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1kbjjd5-MuiContainer-root"
  >
    <nav
      aria-label="breadcrumb"
      class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-1yp6lty-MuiTypography-root-MuiBreadcrumbs-root"
    >
      <ol
        class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
      >
        <li
          class="MuiBreadcrumbs-li"
        >
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
            href="/"
          >
            Home
          </a>
        </li>
        <li
          aria-hidden="true"
          class="MuiBreadcrumbs-separator css-n1sla7-MuiBreadcrumbs-separator"
        >
          <svg>
            chevron-right.svg
          </svg>
        </li>
        <li
          class="MuiBreadcrumbs-li"
        >
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-10eimqc-MuiTypography-root-MuiLink-root"
            data-testid="resources-breadcrumb"
            href=""
          >
            Resources
          </a>
        </li>
        <li
          aria-hidden="true"
          class="MuiBreadcrumbs-separator css-n1sla7-MuiBreadcrumbs-separator"
        >
          <svg>
            chevron-right.svg
          </svg>
        </li>
        <li
          class="MuiBreadcrumbs-li"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-ohe2uh-MuiTypography-root"
            data-testid="credit-forms-breadcrumb"
          >
            Credit / Forms
          </span>
        </li>
      </ol>
    </nav>
  </div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-cnms5y-MuiContainer-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-1jihdd7-MuiPaper-root-MuiCard-root"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-16xaac9-MuiTypography-root"
      >
        Credit / Forms
      </h5>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-1jihdd7-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiBox-root css-hshm0p"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-l1a6io-MuiTypography-root"
        >
          To open a line of credit, please download the Morsco Credit Application below. When complete, either email the application to 
        </p>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
          href="mailto:creditapps@morsco.com."
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-19xfmah-MuiTypography-root"
          >
            creditapps@morsco.com.
          </p>
        </a>
        <p
          class="MuiTypography-root MuiTypography-body1 css-l1a6io-MuiTypography-root"
        >
          , or fax to 817-882-9842. A credit specialist will contact you within 24-48 hours
        </p>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-gic34e-MuiGrid-root"
        >
          <ul
            class="MuiList-root css-1mk9mw3-MuiList-root"
          >
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/MORSCO-Credit-Application-All-Brands-January-2023.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Customer Account Credit Application
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/MORSCO-Credit-Application-All-Brands-Spanish-January-2023.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Solicitud de Cr√©dito
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding css-16fh9v1-MuiListItem-root"
              href="/files/credit-forms/EFT-One-Time-Form-WW-V2.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Electronic Funds Transfer Authorization Form
                  </span>
                </span>
              </div>
            </a>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`CreditForms CreditForms for fortiline should be matching the snapshot on mobile 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg MuiContainer-disableGutters css-b628xg-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-3u2h7w"
      data-testid="back-button-mobile"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
        href="/"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-disableElevation MuiButtonBase-root  css-lw1dhn-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeMedium css-1sx3uk1-MuiButton-startIcon"
          >
            <svg>
              chevron-left.svg
            </svg>
          </span>
          Back to Home
        </button>
      </a>
    </div>
  </div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-cnms5y-MuiContainer-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-18x8ztc-MuiPaper-root-MuiCard-root"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-16xaac9-MuiTypography-root"
      >
        Credit / Forms
      </h5>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-1uovzmr-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiBox-root css-hshm0p"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-l1a6io-MuiTypography-root"
        >
          To open a line of credit, please download the Morsco Credit Application below. When complete, either email the application to 
        </p>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
          href="mailto:creditapps@morsco.com."
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-19xfmah-MuiTypography-root"
          >
            creditapps@morsco.com.
          </p>
        </a>
        <p
          class="MuiTypography-root MuiTypography-body1 css-l1a6io-MuiTypography-root"
        >
          , or fax to 817-882-9842. A credit specialist will contact you within 24-48 hours
        </p>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-gic34e-MuiGrid-root"
        >
          <ul
            class="MuiList-root css-1mk9mw3-MuiList-root"
          >
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/MORSCO-Credit-Application-All-Brands-January-2023.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Customer Account Credit Application
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/MORSCO-Credit-Application-All-Brands-Spanish-January-2023.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Solicitud de Cr√©dito
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding css-16fh9v1-MuiListItem-root"
              href="/files/credit-forms/EFT-One-Time-Form-WW-V2.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Electronic Funds Transfer Authorization Form
                  </span>
                </span>
              </div>
            </a>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`CreditForms CreditForms should be matching the snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1kbjjd5-MuiContainer-root"
  >
    <nav
      aria-label="breadcrumb"
      class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-1yp6lty-MuiTypography-root-MuiBreadcrumbs-root"
    >
      <ol
        class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
      >
        <li
          class="MuiBreadcrumbs-li"
        >
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
            href="/"
          >
            Home
          </a>
        </li>
        <li
          aria-hidden="true"
          class="MuiBreadcrumbs-separator css-n1sla7-MuiBreadcrumbs-separator"
        >
          <svg>
            chevron-right.svg
          </svg>
        </li>
        <li
          class="MuiBreadcrumbs-li"
        >
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-10eimqc-MuiTypography-root-MuiLink-root"
            data-testid="resources-breadcrumb"
            href=""
          >
            Resources
          </a>
        </li>
        <li
          aria-hidden="true"
          class="MuiBreadcrumbs-separator css-n1sla7-MuiBreadcrumbs-separator"
        >
          <svg>
            chevron-right.svg
          </svg>
        </li>
        <li
          class="MuiBreadcrumbs-li"
        >
          <span
            class="MuiTypography-root MuiTypography-caption css-ohe2uh-MuiTypography-root"
            data-testid="credit-forms-breadcrumb"
          >
            Credit / Forms
          </span>
        </li>
      </ol>
    </nav>
  </div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-cnms5y-MuiContainer-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-1jihdd7-MuiPaper-root-MuiCard-root"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-16xaac9-MuiTypography-root"
      >
        Credit / Forms
      </h5>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-1jihdd7-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiBox-root css-hshm0p"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-l1a6io-MuiTypography-root"
        >
          To open a line of credit, please download the Morsco Credit Application below. When complete, either email the application to 
        </p>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
          href="mailto:creditapps@morsco.com."
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-19xfmah-MuiTypography-root"
          >
            creditapps@morsco.com.
          </p>
        </a>
        <p
          class="MuiTypography-root MuiTypography-body1 css-l1a6io-MuiTypography-root"
        >
          , or fax to 817-882-9842. A credit specialist will contact you within 24-48 hours
        </p>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-gic34e-MuiGrid-root"
        >
          <ul
            class="MuiList-root css-1mk9mw3-MuiList-root"
          >
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/MORSCO-Credit-Application-All-Brands-January-2023.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Customer Account Credit Application
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/MORSCO-Credit-Application-All-Brands-Spanish-January-2023.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Solicitud de Cr√©dito
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/INVOICE-AND-DELIVERY-TICKET-Terms-of-Sale_12-29-2014.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Invoice & Delivery Ticket Terms of Sale
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/EFT-Customer-Authorization-Form-Morrison.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Electronic Funds Transfer Authorization Form
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding css-16fh9v1-MuiListItem-root"
              href="/files/credit-forms/Morsco_Tool_Rental_Agreement_Form_20211105.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Equipment Rental Agreement
                  </span>
                </span>
              </div>
            </a>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`CreditForms CreditForms should be matching the snapshot on mobile 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg MuiContainer-disableGutters css-b628xg-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-3u2h7w"
      data-testid="back-button-mobile"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
        href="/"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-disableElevation MuiButtonBase-root  css-lw1dhn-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeMedium css-1sx3uk1-MuiButton-startIcon"
          >
            <svg>
              chevron-left.svg
            </svg>
          </span>
          Back to Home
        </button>
      </a>
    </div>
  </div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-cnms5y-MuiContainer-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-18x8ztc-MuiPaper-root-MuiCard-root"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-16xaac9-MuiTypography-root"
      >
        Credit / Forms
      </h5>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-1uovzmr-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiBox-root css-hshm0p"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-l1a6io-MuiTypography-root"
        >
          To open a line of credit, please download the Morsco Credit Application below. When complete, either email the application to 
        </p>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-fe2j4-MuiTypography-root-MuiLink-root"
          href="mailto:creditapps@morsco.com."
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-19xfmah-MuiTypography-root"
          >
            creditapps@morsco.com.
          </p>
        </a>
        <p
          class="MuiTypography-root MuiTypography-body1 css-l1a6io-MuiTypography-root"
        >
          , or fax to 817-882-9842. A credit specialist will contact you within 24-48 hours
        </p>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-gic34e-MuiGrid-root"
        >
          <ul
            class="MuiList-root css-1mk9mw3-MuiList-root"
          >
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/MORSCO-Credit-Application-All-Brands-January-2023.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Customer Account Credit Application
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/MORSCO-Credit-Application-All-Brands-Spanish-January-2023.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Solicitud de Cr√©dito
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/INVOICE-AND-DELIVERY-TICKET-Terms-of-Sale_12-29-2014.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Invoice & Delivery Ticket Terms of Sale
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding MuiListItem-divider css-14pe0qt-MuiListItem-root"
              href="/files/credit-forms/EFT-Customer-Authorization-Form-Morrison.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Electronic Funds Transfer Authorization Form
                  </span>
                </span>
              </div>
            </a>
            <a
              class="MuiListItem-root MuiListItem-padding css-16fh9v1-MuiListItem-root"
              href="/files/credit-forms/Morsco_Tool_Rental_Agreement_Form_20211105.pdf"
              target="_blank"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg>
                  save-pdf.svg
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-1b7xk9n-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-zvlsvz-MuiTypography-root"
                >
                  <span
                    class="MuiBox-root css-6pr3a7"
                  >
                    Equipment Rental Agreement
                  </span>
                </span>
              </div>
            </a>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;
